### ISSUE TITLE
Error "undefined method `merge' for nil:NilClass"

### ISSUE THREAD

**Issue by brauliobo** [mahipal/jade-rails#1]
On `jade-rails-1.9.2.0/lib/jade/template.rb:34`:

``` ruby
jade_config = context.environment.context_class.jade_config.merge(options)
```

Using rails 4.2.4

**Comment by PandaWhisperer** [comment]
Getting this as well. Also on Rails 4.2.4.

**Comment by brauliobo** [comment]
@PandaWhisperer closing this as I switched to Slim.

**Comment by paulrex** [comment]
Thanks for reporting.

This is fixed in versions >= 1.9.2.1.
